releasenotes/notes/runtime-security-config-file-handling-a9508fda817b9b30.yaml

```diff
@@ -0,0 +1,6 @@
+---
+enhancements:
+  - |
+    The security agent will fail quickly if the first of either the default, or if they exist, user-provided override config files are inaccessible.
+    The default config files are still datadog.yaml and security-agent.yaml, in that order.
+    Previously, if the security agent was started with at least one accessible config file, no matter the order, the agent would still be able to start.
```

Comment:
Can we update this to what was decided on here? https://github.com/DataDog/datadog-agent/pull/14814#pullrequestreview-1261831310

Reply:
sorry about this, this change was stacked on top of the previous change and then i did something weird in git!